jbi/app.py

```diff
@@ -21,13 +19,20 @@
 settings = get_settings()
 version_info = get_version()
 
+
+def traces_sampler(sampling_context: dict[str, Any]) -> float:
+    """Function to dynamically set Sentry sampling rates"""
+
+    request_path = sampling_context.get("asgi_scope", {}).get("path")
+    if request_path == "/__lbheartbeat__":
+        # Drop all __lbheartbeat__ requests
+        return 0
+    return settings.sentry_traces_sample_rate
+
+
 sentry_sdk.init(
     dsn=settings.sentry_dsn,
-    integrations=[
-        StarletteIntegration(),
-        FastApiIntegration(),
-    ],
-    traces_sample_rate=settings.sentry_traces_sample_rate,
+    traces_sampler=traces_sampler,
```

Comment:
I think I prefer the  defaulted "url" vs the "endpoint"; so nothing to change here--if we wanted to modify the transaction name presented in sentry in the future, we could look to this anchor (of the link you provided) https://docs.sentry.io/platforms/python/guides/fastapi/#integration-options

Reply:
_(resolving this comment to unblock)_